src/main/scala/wdl4s/UnsatisfiedInputsException.scala

```diff
@@ -1,5 +1,18 @@
 package wdl4s
 
+import scalaz.NonEmptyList
+
+trait ThrowableWithErrors extends Throwable {
+  def message: String
+  def errors: NonEmptyList[String]
+
+  override def getMessage = {
+    s"""$message\n${errors.list.mkString("\n")}"""
+  }
+}
+
 class UnsatisfiedInputsException(message: String, cause: Throwable) extends RuntimeException(message, cause) {
   def this(message: String) = this(message, null)
 }
+
+class RuntimeValidationException(val message: String, val errors: NonEmptyList[String]) extends RuntimeException(message) with ThrowableWithErrors
```

Comment:
Yes those cases use the pre-existing `UnsatisfiedInputsException`, which is actually correct because they're thrown when a input is missing or cannot be coerced.